Marketplace/musicVisualizer/musicVisualizer.js

```diff
@@ -0,0 +1,293 @@
+//  
+//  musicVisualizer.js
+//  A tablet app for spawning particle entities with audio reactivity
+//   
+//  Author: Elisa Lupin-Jimenez
+//  Copyright High Fidelity 2017
+//  
+//  Licensed under the Apache 2.0 License
+//  See accompanying license file or http://apache.org/
+//  
+//  All assets are under CC Attribution Non-Commerical
+//  http://creativecommons.org/licenses/
+//  
+
+var LIB = Script.require("./musVisLib.js?" + Date.now());
+var MIC_SYNC_SCRIPT = Script.resolvePath("adjuster_scripts/micSync.js?" + Date.now());
+var AUDIO_SYNC_SCRIPT = Script.resolvePath("adjuster_scripts/audioFileSync.js?" + Date.now());
+var TRAIL_LEFT_SCRIPT = Script.resolvePath("adjuster_scripts/effectTrailerLeft.js?" + Date.now());
+var TRAIL_RIGHT_SCRIPT = Script.resolvePath("adjuster_scripts/effectTrailerRight.js?" + Date.now());
+
+var CLEAR_SELECTION_TEXT = "Clear selection";
+
+(function() {
+
+    var APP_NAME = "MUS VIS";
+    var APP_URL = "https://hifi-content.s3.amazonaws.com/elisalj/music_visualizer/musicVisualizerUI.html?" + Date.now();
```

Comment:
We probably want this to be relative, so it's referencing the new HTML with cleanup button